CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,200 @@
+regulations on this matter.
+
+Repeated violations of this policy are grounds for enforcement actions as if
+against unacceptable behaviour.
+
+## Reporting guidelines
+
+If you are subject to, or witness, unacceptable behaviour, or have any other
+concerns, please notify a community organizer as soon as possible. The current
+list of community organizers is:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+
+When reporting, please include the following information, to the best of your
+knowledge and ability:
+
+* When and where the unacceptable behaviour occurred.
+* Who engages in unacceptable behaviour.
+* Which unacceptable behaviour(s) happened.
+* Any evidence for your report (such as screencaps, logs, etc)
+* Who (besides yourself) was affected by this behaviour.
+* Whether you want any additional actions taken by the community organizers to 
+  ensure that your safety and well-being remain intact.
+
+Additionally, community organizers can assist local authorities and connect you
+with help or providing a safe environment. In the context of in-person events, 
+organizers will also provide escorts as desired by the person experiencing distress.
+
+## Addressing grievances
+
+If you feel you have been falsely or unfairly accused of violating this Code of
+Conduct, you should notify the community organizers with a concise
+description of your grievance. You should include the following information:
+
+* What report of unacceptable behaviour you are responding to.
+* Why you believe specifically that you did not engage in this behaviour, or
+  why you believe the enforcement actions taken against you are
+  disproportionate.
+* Any evidence you can provide for your grievance.
+* What actions you wish to be taken to address your grievance if found genuine.
+
+If you have been temporarily, or permanently, banned without right of appeal,
+any grievances filed with community organizers regarding the incident which
+caused the ban will be ignored.
+
+## Scope
+
+We expect all community participants (contributors, paid or otherwise; sponsors;
+and other guests) to abide by this Code of Conduct in all 
+community venues, as well as in one-to-one communications pertaining to community 
+business. This Code of Conduct also applies to unacceptable behaviour occuring 
+outside the scope of community activities when such behaviour has the 
+potential to adversely affect the safety and well-being of community members.
+
+## Contact information
+
+This is the current list of community organizers, with appropriate contact
+information:
+
+* Sylvia van Os <sylvia@hackerchick.me> (lead organizer)
+
+## Modifications
+
+The community organizers reserve the right to modify this Code of Conduct to 
+take into account future situations that could not have been foreseen
+at the time of its writing, or to address lack of clarity or inconsistencies
+that become apparent with practice. Additionally, we must update this document
```

Comment:
"must" sounds weird. Why not just "will"?

Reply:
I'm trying to construct a justification for requiring an update schedule in the first place, and needing to fix the document in case of changes to the enforcement team is one such justification. Hence my use of 'must', in the sense of 'we have no choice but to'.

Reply:
I think "must" just sounds weird. It implies we'd rather not. I'd just say "we will", it's a promise and it shows that we care to do so.

Reply:
I'll just redraft this whole section to cleanly separate 'why we need this' from 'what we will do about it', because I think this is the heart of this problem.